about.html

```diff
@@ -0,0 +1,198 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Portfolio Design" />
+    <meta name="author" content="Adeyemi Dada" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link href="assets/css/style.css" rel="stylesheet">
+    <title>Global Impact Youth Summit 2023</title>
+</head>
+
+<body>
+
+    <header>
+
+        <div class="header-top">
+            <div class="container">
+                <ul>
+                    <li><a href="#"><img src="./assets/images/facebook.png" alt="facebook" /></a></li>
+                    <li><a href="#"><img src="./assets/images/twitter.png" alt="facebook" /></a></li>
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">Logout</a></li>
+                </ul>
+            </div>
+        </div>
+
+
+
+        <nav class="navbar navbar-expand-lg navbar-light sticky-top">
+            <div class="container">
+                <a class="navbar-brand" href="index.html">
+                    <img src="./assets/images/logo.png" alt="logo" />
+                </a>
+                <button id="mobile-menu-btn" class="navbar-toggler" type="button">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                    <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+                        <li class="nav-item">
+                            <a class="nav-link active" aria-current="page" href="about.html">About</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Program</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Join</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Sponsor</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">News</a>
+                        </li>
+                        <li class="nav-item">
+                            <button type="button" class="btn btn-outline-primary">Impart Campaign</button>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+
+        <div class="popup-menu">
+            <img class="popup-close" id="popup-close-btn" src="./assets/images/popup-close-white.svg" />
+            <ul>
+                <li class="popup-menu-item">
+                    <a class="nav-link" href="about.html">About</a>
```

Comment:
[ ] kindly Fix your navbar, so It's possible to navigate between the pages, there's a link from the Home page to the About page and the opposite too.🤓
your design: 👇
![about](https://github.com/yemidada/captone-project/assets/62156607/3a862bb6-b5eb-46f1-9b30-d70e35936924)

Figma design: 👇
![design](https://github.com/yemidada/captone-project/assets/62156607/08d22c7c-837a-4167-84b4-f18ff07abacd)